tests/Unit/Migration/Pdk/Migration5_0_0Test.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\WooCommerce\Migration\Pdk;
+
+use GuzzleHttp\Psr7\Response;
+use MyParcelNL\Pdk\Facade\Logger;
+use MyParcelNL\Pdk\Facade\Pdk;
+use MyParcelNL\Pdk\Facade\Settings;
+use MyParcelNL\Pdk\Settings\Contract\SettingsRepositoryInterface;
+use MyParcelNL\Pdk\Settings\Model\AccountSettings;
+use MyParcelNL\Pdk\Tests\Bootstrap\MockApi;
+use MyParcelNL\Pdk\Tests\Bootstrap\MockSettingsRepository;
+use MyParcelNL\WooCommerce\Migration\Migration5_0_0;
+use MyParcelNL\WooCommerce\Tests\Mock\WordPressOptions;
+use MyParcelNL\WooCommerce\Tests\Uses\UsesMockWcPdkInstance;
+use function DI\autowire;
+use function MyParcelNL\Pdk\Tests\usesShared;
+
+usesShared(
+    new UsesMockWcPdkInstance([
+        SettingsRepositoryInterface::class => autowire(MockSettingsRepository::class)->constructor([
+            AccountSettings::ID => [
```

Comment:
niet zo doen, gebruik factories (dit is oud, achterhaald en werkt straks niet meer)